test_util/cluster.py

```diff
@@ -323,15 +323,14 @@ def install_dcos(
 def upgrade_dcos(cluster, installer_url, add_config_path=None):
 
     def upgrade_host(tunnel, role, bootstrap_url):
-        # Download the install script for the new DC/OS.
-        tunnel.remote_cmd(curl_cmd + ['--remote-name', bootstrap_url + '/dcos_install.sh'])
+        # Download the upgrade script for the new DC/OS.
+        tunnel.remote_cmd(curl_cmd + ['--remote-name', bootstrap_url + '/upgrade.latest'])
+        upgrade_id = tunnel.remote_cmd(['cat', 'upgrade.latest'])
```

Comment:
This test needs to reflect how our upgrade docs will look once published. According to the design doc, we'll be asking the user to get the URL from the installer's stdout, so this test will need to get that URL the same way.